.stylelintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": ["stylelint-config-standard-scss"]
+  "extends": ["stylelint-config-standard-scss"],
+  "rules": {
+    "no-invalid-position-at-import-rule": null
```

Comment:
We're nesting `@import` in `.seam-components`, so I'm not sure this is required?

Reply:
I wonder if we can use `@use` instead and make writing all the styles a bit easier. Docs suggest (1) `@import` is going to be removed and (2) `@use` can namespace so we don't need to prefix all the styles, Sass will do that for us.

Reply:
https://github.com/seamapi/react/discussions/81